src/renderer/components/sponsor-block-settings/sponsor-block-settings.js

```diff
@@ -46,6 +46,12 @@ export default defineComponent({
     useDeArrowThumbnails: function () {
       return this.$store.getters.getUseDeArrowThumbnails
     },
+    deArrowShowOriginal: function () {
+      return this.$store.getters.getDeArrowShowOriginal
+    },
+    deArrowShowOriginalAlwaysOn: function () {
+      return this.$store.getters.getDeArrowShowOriginalAlwaysOn
+    },
```

Comment:
It looks like this value doesn't exists in the store and is also not used in the component. This is presumably a leftover from previous iterations of this pull request?

Reply:
Yeah that is leftover, my bad